Sources/MongoSwift/MongoClient.swift

```diff
@@ -14,6 +14,10 @@ public struct MongoClientOptions: CodingStrategyProvider {
     /// - SeeAlso: https://docs.mongodb.com/manual/reference/connection-string/#urioption.compressors
     public var compressors: [Compressor]?
 
+    /// Specifies the time, in milliseconds, for an individual connection from your connection pool to establish a TCP
+    /// connection to the MongoDB server before timing out.
```

Comment:
Some docs nits:
- can we add "maximum" before "time" here?
- can we change "the" (in "the MongoDB server") to "a"?
- this value actually also applies to monitoring connections (i.e. connections not included in the pools), so can we drop the "from your connection pool"?